rollup-cli/src/actions/offchain/send.js

```diff
@@ -43,7 +43,7 @@ async function send(UrlOperator, idTo, amount, walletJson, password, tokenId, us
                 };
                 walletRollup.signRollupTx(transaction); // sign included in transaction
                 const parsetransaction = stringifyBigInts(transaction);// convert bigint to Strings
```

Comment:
We should use lowerCamelCase naming convention:
`parsetransaction` to `parseTransaction`